### PR TITLE
Add missing styleWrapper style builder information in container/Grid

### PR DESCRIPTION
**Pull request by sneridagh**
This PR fixes the missing style information for inner container/grid blocks.
Adds a schemaEnhancer for disabling Teaser align handlers for inner container teaser elements.

### CODE REVIEW

File: packages/volto/src/components/manage/Blocks/Teaser/schema.js [hunk 1]
```diff
@@ -127,3 +127,15 @@ export const gridTeaserDisableStylingSchema = ({ schema, formData, intl }) => {
...
+  schema,
+  formData,
+  intl,
+}) => {
+  schema.properties.styles.schema.fieldsets[0].fields =
+    schema.properties.styles.schema.fieldsets[0].fields.filter(
+      (item) => !['align'].includes(item),
+    );
+  return schema;
+};
```
Comment (davisagli) [inline packages/volto/src/components/manage/Blocks/Teaser/schema.js:141]: @sneridagh Why is this here, but not used?

Comment (sneridagh) [reply packages/volto/src/components/manage/Blocks/Teaser/schema.js:141]: Do you think this should be only in VLT?

**Review by davisagli — APPROVED** [review]
